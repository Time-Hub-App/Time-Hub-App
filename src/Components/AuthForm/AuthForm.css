Button {
  color: black;
}

input {
  border-radius: 100px;
}
