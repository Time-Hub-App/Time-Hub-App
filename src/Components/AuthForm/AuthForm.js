import React from 'react';

export default function AuthForm({
  email,
  setEmail,
  username,
  setUsername,
  password,
  setPassword,
  errorMsg,
  handleSubmit
}) {
  return (
    <form className="form" onSubmit={handleSubmit}>
      <div>
        <label>Email:</label>
        <input type="email" value={email} onChange={(e) => setEmail(e.target.value)} />
      </div>
      <div>
        <label>Username:</label>
        <input type="text" value={username} onChange={(e) => setUsername(e.target.value)} />
      </div>
      <div>
        <label>Password:</label>
        <input type="password" value={password} onChange={(e) => setPassword(e.target.value)} />
      </div>
      <p>{errorMsg}</p>
      <input type="submit" />
    </form>
  );
}
