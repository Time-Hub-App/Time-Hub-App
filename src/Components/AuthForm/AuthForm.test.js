import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import AuthForm from './AuthForm';

it('should render a header', () => {
  const { container } = render(
    <MemoryRouter>
      <AuthForm />
    </MemoryRouter>
  );

  expect(container).toMatchSnapshot();
});
