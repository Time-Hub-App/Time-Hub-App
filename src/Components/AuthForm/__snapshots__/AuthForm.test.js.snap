// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a header 1`] = `
<div>
  <form
    class="form"
  >
    <label>
      Email: 
    </label>
    <input
      type="email"
      value=""
    />
    <label>
      Password: 
    </label>
    <input
      type="password"
      value=""
    />
    <p />
    <input
      type="submit"
    />
  </form>
</div>
`;
